tests/fixtures/express/server.ts

```diff
@@ -21,14 +21,16 @@ import '../controllers/tagController';
 import { RegisterRoutes } from './routes';
 
 export const app: express.Express = express();
-app.use(bodyParser.urlencoded({ extended: true }));
-app.use(bodyParser.json());
-app.use(methodOverride());
-app.use((req: any, res: any, next: any) => {
+export const router = express.Router();
```

Comment:
Move this to a new fixture, something like express-router/server.ts, leave the old one as is

Reply:
My initial concern was that it adds a bunch of work to the test suite for a very small check.  For now I've created an "express-router" test case copied from the "express" test case.  But maybe it can be reduced?

Or maybe I can should just change the "express-openapi3" test to use `express.Router`?

Reply:
The amount of specs is excessive, I agree. But we should have a separate fixture.
The specs should be something like [inversify-glob.spec.ts.](https://github.com/lukeautry/tsoa/blob/master/tests/integration/inversify-glob.spec.ts) Just 1-2 tests ensuring the route works.

Reply:
If you revert the changes here, I  think we should be good to go.

Reply:
Ah whoops, sorry, not sure how I missed this.